jwst/ami/instrument_data.py

```diff
@@ -109,14 +115,14 @@ def read_data_model(self, input_model):
 
         Parameters
         ----------
-        input_model : instance Data Model
+        input_model : `~stdatamodels.DataModel`
```

Comment:
Did we decide yet whether these should all point to the datamodel base class or to JwstDataModel?

Reply:
I agree it's out of scope, but it might save someone some time if we already knew.  I'm happy with it as-is

Reply:
I will just change this one to JwstDataModel now while I am at it. Do you want a new GH issue to discuss the package wide question?

Reply:
That would not hurt, but IMO it's not very high priority either